soweego/importer/imdb_dump_extractor.py

```diff
@@ -0,0 +1,390 @@
+        person_entity.name = person_info.get("primaryName")
+        person_entity.name_tokens = " ".join(
+            text_utils.tokenize(person_entity.name))
+
+        # If either `actor` or `actress` in primary profession
+        # (which is a comma separated string of professions)
+        # then we can distinguish the gender
+        if any(prof in person_info.get("primaryProfession")
+               for prof in ["actor", "actress"]):
+            person_entity.gender = "male" if "actor" in person_info.get(
+                "primaryProfession") else "female"
+
+        # datetime.date(year, month, day)
+        born_year = person_info.get("birthYear")
+        if born_year:
+            person_entity.born = datetime.date(int(born_year), 1, 1)
+
+        death_year = person_info.get("deathYear")
+        if death_year:
+            person_entity.died = datetime.date(int(death_year), 1, 1)
+
+        # The array of primary professions gets translated to a list
+        # of the QIDs that represent said professions in Wikidata
+        if person_info.get("primaryProfession"):
+            person_entity.occupations = " ".join(self._translate_professions(
+                person_info.get("primaryProfession").split(",")
+            ))
+
+        session.add(person_entity)
+
+    def _populate_person_movie_relations(self, person_info: Dict,
+                                         session: object) -> None:
+        """
+        Given a `person_info` we extract the ID that the person has
+        in IMDB and the IDs of the movies for which this person is
+        known (which also come from IMDB). We add a
+        :ref:`soweego.importer.models.imdb_entity.ImdbPersonMovieRelationship`
+        entity to the session for each realtion.
+
+        :param person_info: dictionary that contains the IMDB person ID and
+        the IMDB movie IDs (for movies the specific person is known). The movie
+        IDs are a comma separated string
+        :param session: the SQLAlchemy session to which we will
+        add the relation entities.
+        """
+
+        know_for_titles = person_info.get(
+            "knownForTitles").split(",")
+
+        for title in know_for_titles:
+
+            session.add(imdb_entity.ImdbPersonMovieRelationship(
+                from_catalog_id=person_info.get("nconst"),
+                to_catalog_id=title
+            ))
+
+    def _translate_professions(self, professions: List[str]) -> List[str]:
+        """
+        Gets the list of professions (as a list of strings) directly from IMDB
+        and translates these to a list of Wikidata QIDs for each specific
+        profession. Unmappable professions (like `miscellaneous` are removed)
+
+        The actual QIDs can be found in
+        :ref:`soweego.wikidata.vocabulary`
+
+        :param professions: list of profession names, given by IMDB
+
+        :return: list of QIDs for said professions
+        """
+        qids = []
+
+        mappings = {
```

Comment:
Please move this mapping to `vocabulary.py` for better retrieval and readability

Reply:
This has been resolved in ddfda7e1f72e156686a456cdf629b2a5922622f9